src/test-func/groovy/org/transmartproject/batch/highdim/proteomics/platform/ProteomicsPlatformCleanScenarioTests.groovy

```diff
@@ -28,14 +33,36 @@ class ProteomicsPlatformCleanScenarioTests implements JobRunningTestTrait {
     static final long NUMBER_OF_PROBES = 8
 
     @ClassRule
-    public final static RunJobRule RUN_JOB_RULE = new RunJobRule(PLATFORM_ID, 'proteomics_annotation')
+    public final static TestRule RUN_JOB_RULES = new RuleChain([
+            new RunJobRule(PLATFORM_ID, 'proteomics_annotation'),
+            new ExternalResource() {
+                @Override
+                protected void before() throws Throwable {
+                    super.before()
+
+                    NamedParameterJdbcTemplate jdbcTempleate = new AnnotationConfigApplicationContext(
+                            GenericFunctionalTestConfiguration).getBean(NamedParameterJdbcTemplate)
```

Comment:
This will leak a connection. Use the static application context (or close the application context after you use it).